app/components/file-details/dynamic-scan/drawer-old/proxy-settings-view/index.ts

```
@@ -0,0 +1,95 @@
+        );
+      } else {
+        triggerAnalytics(
+          'feature',
+          ENV.csb['disableProxy'] as CsbAnalyticsFeatureData
+        );
+      }
+    } catch (error) {
+      const err = error as AdapterError;
+      let errMsg = this.intl.t('plaseTryAgain');
+
+      if (err.errors && err.errors.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)